### PR TITLE
Fixing minor typos

### PR DESCRIPTION
**Pull request by aarongustafson**

### CODE REVIEW

File: index.md [hunk 1]
```diff
@@ -15,7 +15,7 @@
...
+We've always thought there's not enough Developer Avocados out there, and we wanted to help change that. When we started out, we used to read [Technically Speaking](https://techspeak.email/) a lot, but Cate and Chiu-Ki took a break. We've waited this past year for someone to pick up the slack and come up with a replacement. Turns out, it's us 😅! We want to keep a similar minimalist format, with a few highly curated resources and a few good conference CFPs. Delivered every  week in your inbox. But we are changing the focus from strictly public  speaking. Our resources and tips geared towards helping you become a (better) Developer Avocado 🥑!
 
 So Developer Avocados 🥑 Weekly is a curated list of articles, resources and CFPs. It saves you time and all the FOMO of missing out. 💌 You could just read the [archive](https://tinyletter.com/developeravocados/archive), but we feel this weekly email in your inbox is going to keep reminding you there is a world of avocados out there, just like you, so please subscribe 💚. We'd appreciate you spreading the word on social media if you like what you see.
```
Comment (aarongustafson) [inline index.md]: On a side not, I’d really like to change "FOMO of missing out" to an `abbr` of "FOMO" with an `aria-label` of "Fear Of Missing Out". Thoughts?

Comment (AlexLakatos) [reply index.md]: Yes please.

Comment (aarongustafson) [reply index.md]: Done.

**Comment by AlexLakatos** [comment]
Thank you!

**Pull request merged**